azurerm/data_source_key_vault.go

```diff
@@ -177,7 +177,7 @@ func dataSourceArmKeyVaultRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("enabled_for_template_deployment", props.EnabledForTemplateDeployment)
 		d.Set("vault_uri", props.VaultURI)
 
-		if err := d.Set("sku", flattenKeyVaultDataSourceSku(props.Sku)); err != nil {
+		if err := d.Set("sku_name", flattenKeyVaultDataSourceSku(props.Sku)); err != nil {
```

Comment:
Is this a mistake? there is no property `sku_name` in the schema for this resource

Reply:
No, it is not a mistake... the property coming from Azure will still be props.Sku, so I need to read that and then convert it to the new property `sku_name`